lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -850,17 +850,15 @@ public function getLowerExpression($str)
     }
 
     /**
-     * Returns the SQL snippet to get the position of the first occurrence of substring $substr in string $str.
+     * Returns the SQL snippet to get the position of the first occurrence of the substring in the string.
      *
-     * @param string    $str      Literal string.
-     * @param string    $substr   Literal string to find.
-     * @param int|false $startPos Position to start at, beginning of string by default.
-     *
-     * @return string
+     * @param string      $string    String to locate the substring in, SQL expression.
+     * @param string      $substring Substring to locate, SQL expression.
```

Comment:
I think a better naming could be "needle"

Reply:
I'd keep it closer to the SQL naming, not to PHP. Otherwise, it'd have to be `$haystack` and `$needle`.